### ISSUE TITLE
does it work for Europe country numbers?

### ISSUE THREAD

**Issue by profesionalleet1337** [rafasantos/twilio-sms-web#8]
Hello, does this will work for Europe country numbers ? And does it allow to put link's/ url in the message box ?

**Comment by rafacandev** [comment]
> Hello, does this will work for Europe country numbers ?
Yes.

For clarification, this application is a web user interface for Twilio and you can use any phone number purchased at Twilio for sending and reading messages. Twilio offers coverage in many countries, I suggest you to search on Twilio's website and check if they cover the countries you want to send/receive messages. For example, here is the coverage for Spain: https://www.twilio.com/en-us/sms/pricing/es

> And does it allow to put link's/ url in the message box ?

This application has no limitations on the content that you send or receive and you can send and receive links/urls. However, keep in mind that you need to comply with the rules and regulations for the countries you are operating. Moreover, many carriers may block messages with links/urls because they may interpret these messages as spam.

Here is an introductory article about message filtering:
https://support.twilio.com/hc/en-us/articles/1260803966670-How-do-I-prevent-my-Twilio-messages-from-being-filtered-blocked-

**Comment by profesionalleet1337** [comment]
> For clarification, this application is a web user interface for Twilio and you can use any phone number purchased at Twilio for sending and reading messages. Twilio offers coverage in many countries, I suggest you to search on Twilio's website and check if they cover the countries you want to send/receive messages. For example, here is the coverage for Spain: https://www.twilio.com/en-us/sms/pricing/es
> 
> > And does it allow to put link's/ url in the message box ?
> 
> This application has no limitations on the content that you send or receive and you can send and receive links/urls. However, keep in mind that you need to comply with the rules and regulations for the countries you are operating. Moreover, many carriers may block messages with links/urls because they may interpret these messages as spam.
> 
> Here is an introductory article about message filtering: https://support.twilio.com/hc/en-us/articles/1260803966670-How-do-I-prevent-my-Twilio-messages-from-being-filtered-blocked-

Thanks for the detailed explanation. So to use it, I have to "purchase" a phone number from Twilio, If I understand you correct ? There is no free option ?

**Comment by rafacandev** [comment]
> So to use it, I have to "purchase" a phone number from Twilio, If I understand you correct ? There is no free option ?

That is correct. You could get a free trial phone number at Twilio but it has very limited usage.

**Comment by profesionalleet1337** [comment]
> > So to use it, I have to "purchase" a phone number from Twilio, If I understand you correct ? There is no free option ?
> 
> That is correct. You could get a free trial phone number at Twilio but it has very limited usage.

Thank you for your reply ! How I could eventually take/have free number , so I could try ?

**Comment by rafacandev** [comment]
Here is the documentation for a free trial: https://www.twilio.com/docs/usage/tutorials/how-to-use-your-free-trial-account

Keep in mind, that there are limitations (scroll down the page and check out the *Trial Account restrictions and limitations* session).

**Comment by rafacandev** [comment]
Closed due to inactivity.
